package.json

```diff
@@ -73,6 +73,7 @@
     "react-router-dom": "^5.0.1",
     "react-sortable-hoc": "^1.10.1",
     "react-sortable-tree": "^2.6.2",
+    "saleor-cli": "^0.10.0",
```

Comment:
Should we change this? Just asking

Reply:
not really, i see it was added automatically to my environment because I'm working with saleor-cli tool